### PR TITLE
Enforce budget on cost estimate validation

### PR DESCRIPTION
**Pull request by lepistone**
This work is intended to bring the enforcement of the budget to the cost estimate validation.

Because cost estimates are... estimates, it is too strict to enforce the budget on the creation (that was disabled already). Instead, we check when the estimate is validated into a logistic order.
- [x] rebase on top of #46 
- [x] check the views
- [x] update the readme

### CODE REVIEW

File: logistic_budget/README.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Logisitic budget
+================
+
+This module adds a notion of budget on logistic requests and cost estimates.
+
+A line of a logistic request can have a budget and a pre-approval by a budget
+holder and a finance officer.
+
+The creation of a cost estimate can be done without approval of with a budget
```
Comment (gurneyalex) [inline logistic_budget/README.rst]: "of with" -> parse error.

File: logistic_budget/README.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+overrun. On the other hand, approving a cost estimate into a logistic order
+required approval by the budget holder, and the limit must be respected.
```
Comment (gurneyalex) [inline logistic_budget/README.rst]: s/required/requires/ ?

**Comment by gurneyalex** [comment]
nice test refactoring  work. :+1: once the minor things in the README are fixed.

**Comment by lepistone** [comment]
readme fixed

**Comment by foutoucour** [comment]
:+1:

File: logistic_budget/README.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Logisitic budget
+================
```
Comment (guewen) [inline logistic_budget/README.rst]: s/Logisitic/Logistics/

Comment (lepistone) [inline logistic_budget/README.rst]: done

**Comment by lepistone** [comment]
Note: travis red build seems unrelated: pylint crashes with `AssertionError: Message symbol 'print-statement' is already defined`.

The last commit changes only the readme, the one before is green.

**Comment by guewen** [comment]
:+1:

**Pull request merged**